{"uid":"9c91a5e040cb8cfd","name":"test_build_and_deploy","fullName":"tests.integration.test_node_port#test_build_and_deploy","historyId":"8e5b0ff388919a946524d5696c2557e5","time":{"start":1731807437486,"stop":1731808342531,"duration":905045},"description":"Test the deployment of the charm.","descriptionHtml":"<p>Test the deployment of the charm.</p>\n","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_node_port.py\", line 129, in test_build_and_deploy\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1731807437263,"stop":1731807437263,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1731807437263,"stop":1731807437263,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1731807437263,"stop":1731807437485,"duration":222},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1731807437485,"stop":1731807437485,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Test the deployment of the charm.","status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","statusTrace":"Traceback (most recent call last):\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: Optional[TResult] = func()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 241, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 87, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 63, in thread_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 90, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 65, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 850, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 833, in _runtest_for\n    yield\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 878, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 183, in pytest_runtest_call\n    raise e\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 173, in pytest_runtest_call\n    item.runtest()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1632, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 162, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/tests/integration/test_node_port.py\", line 129, in test_build_and_deploy\n    await ops_test.model.wait_for_idle(\n  File \"/home/runner/work/mysql-router-k8s-operator/mysql-router-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","steps":[],"attachments":[{"uid":"7907a98dd51410a0","name":"log","source":"7907a98dd51410a0.txt","type":"text/plain","size":8025}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1731808342593,"stop":1731808342593,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1731808342604,"stop":1731808343287,"duration":683},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1731808343288,"stop":1731808343288,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1731808343288,"stop":1731808343288,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1731808343288,"stop":1731808343288,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.skipif(False, reason='Skips juju <3.1.x as we have a dependency for self-signed-certificates')"},{"name":"tag","value":"abort_on_fail"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration"},{"name":"suite","value":"test_node_port"},{"name":"host","value":"fv-az740-536"},{"name":"thread","value":"13210-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.test_node_port"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"6eb6f51b36fc03ae","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":68,"skipped":0,"passed":99,"unknown":2,"total":169},"items":[{"uid":"cb6f827cf6a81d26","reportUrl":"../2036//#testresult/cb6f827cf6a81d26","status":"passed","time":{"start":1731721257410,"stop":1731721493896,"duration":236486}},{"uid":"c284985fe328690c","reportUrl":"../2034//#testresult/c284985fe328690c","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1731634723476,"stop":1731635628704,"duration":905228}},{"uid":"22dd216552ae51b9","reportUrl":"../2033//#testresult/22dd216552ae51b9","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1731548312619,"stop":1731549218010,"duration":905391}},{"uid":"1e632dfe1faffca2","reportUrl":"../2032//#testresult/1e632dfe1faffca2","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1731461769975,"stop":1731462675539,"duration":905564}},{"uid":"4cfa704b8064c9e7","reportUrl":"../2030//#testresult/4cfa704b8064c9e7","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1731375630135,"stop":1731376535666,"duration":905531}},{"uid":"bec52215986dc464","reportUrl":"../2027//#testresult/bec52215986dc464","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1731289146822,"stop":1731290051863,"duration":905041}},{"uid":"33c130f5e993bbeb","reportUrl":"../2026//#testresult/33c130f5e993bbeb","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1731202611541,"stop":1731203516211,"duration":904670}},{"uid":"e8859148d7c59289","reportUrl":"../2025//#testresult/e8859148d7c59289","status":"broken","statusDetails":"juju.errors.JujuAppError: App in error: mysql-k8s","time":{"start":1731116358368,"stop":1731116394578,"duration":36210}},{"uid":"bc7139fe1d17a7b7","reportUrl":"../2024//#testresult/bc7139fe1d17a7b7","status":"unknown","time":{}},{"uid":"d85ac5eb7a341357","reportUrl":"../2022//#testresult/d85ac5eb7a341357","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1730943670139,"stop":1730944575404,"duration":905265}},{"uid":"b496028cb2afd0fc","reportUrl":"../2021//#testresult/b496028cb2afd0fc","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1730857105560,"stop":1730858012915,"duration":907355}},{"uid":"f00a6724b3c0d5ae","reportUrl":"../2019//#testresult/f00a6724b3c0d5ae","status":"passed","time":{"start":1730770803203,"stop":1730771028159,"duration":224956}},{"uid":"3b1f68dbc9dc9d11","reportUrl":"../2017//#testresult/3b1f68dbc9dc9d11","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: Primary\n  mysql-k8s/2 [executing] active:","time":{"start":1730684381705,"stop":1730685287715,"duration":906010}},{"uid":"7dc15023687d595b","reportUrl":"../2016//#testresult/7dc15023687d595b","status":"passed","time":{"start":1730598107397,"stop":1730598325233,"duration":217836}},{"uid":"f65e0fb57379f4b2","reportUrl":"../2015//#testresult/f65e0fb57379f4b2","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1730511578228,"stop":1730512483259,"duration":905031}},{"uid":"537d5e839bc6304a","reportUrl":"../2014//#testresult/537d5e839bc6304a","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1730425704082,"stop":1730426609431,"duration":905349}},{"uid":"bd59675c5aabcc1d","reportUrl":"../2013//#testresult/bd59675c5aabcc1d","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1730338709495,"stop":1730339614864,"duration":905369}},{"uid":"1502520b993d4862","reportUrl":"../2011//#testresult/1502520b993d4862","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: Primary\n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active:","time":{"start":1730252480891,"stop":1730253386967,"duration":906076}},{"uid":"22db8fb4a1e74884","reportUrl":"../2010//#testresult/22db8fb4a1e74884","status":"passed","time":{"start":1730166006208,"stop":1730166230661,"duration":224453}},{"uid":"34f81d8b8efbb3f8","reportUrl":"../2009//#testresult/34f81d8b8efbb3f8","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\nmysql-k8s/0 [executing] active: \n  mysql-k8s/1 [executing] active: \n  mysql-k8s/2 [executing] active: Primary","time":{"start":1730079457979,"stop":1730080364388,"duration":906409}}]},"tags":["asyncio","abort_on_fail","@pytest.mark.skipif(False, reason='Skips juju <3.1.x as we have a dependency for self-signed-certificates')","group(1)"]},"source":"9c91a5e040cb8cfd.json","parameterValues":[]}